### PR TITLE
Sc 1470

### PR DESCRIPTION
**Pull request by amit-tarento**
File changed (Remaining files got re fromatted)
1.FeedController.java
2. routes

### CODE REVIEW

**Comment by rjshrjndrn** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/sunbird-lms-service/sunbird-lms-service/pullRequest?prid=4541941&bid=15285837)

File: service/app/controllers/feed/FeedController.java [hunk 1]
```diff
@@ -1,58 +1,51 @@
...
-
-import java.util.*;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.CompletionStage;
 
 public class FeedController extends BaseController {
 
+  public CompletionStage<Result> feed(String userId, Http.Request httpRequest) {
+    return handleRequest(
+        "getUserFeedById", null, null, userId, JsonKey.USER_ID, false, httpRequest);
```
Comment (manzarul) [inline service/app/controllers/feed/FeedController.java]: use constant String here

File: service/app/controllers/feed/FeedController.java [hunk 2]
```diff
@@ -1,58 +1,51 @@
...
-        Response response=new Response();
-        response.setResponseCode(ResponseCode.OK);
-        response.put(JsonKey.RESPONSE,prepareResponse((int)(Math.random() * 50 + 1),userId));
-        CompletableFuture<JsonNode> future = new CompletableFuture<>();
-        future.complete(Json.toJson(response));
-        return future.thenApplyAsync(Results::ok);
-    }
-
-    private Map<String,Object> prepareResponse(int randVal,String userId){
+  private Map<String, Object> prepareResponse(int randVal, String userId) {
```
Comment (manzarul) [inline service/app/controllers/feed/FeedController.java]: if this is integrated this method can be removed.

**Comment by rjshrjndrn** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity decreasing per file
==============================
+ service/app/controllers/feed/FeedController.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/sunbird-lms-service/sunbird-lms-service/pullRequest?prid=4541941&bid=15285837)

**Pull request closed without merging**